components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41.java

```diff
@@ -441,7 +441,7 @@ public Response removeCartridge(
     @AuthorizationAction("/permission/protected/manage/addServiceGroup")
     @SuperTenantService(true)
     public Response addServiceGroup(
-            GroupBean serviceGroupDefinition) throws RestAPIException {
+            GroupBean serviceGroupDefinition) throws RestAPIException, InvalidCartridgeGroupDefinitionException {
```

Comment:
Yes agreed. Removed the throwing of InvalidCartridgeGroupDefinitionException from the API method.